api/query.py

```diff
@@ -118,8 +118,8 @@ async def twitterOverview(self, asaID: str) -> TwitterOverview:
             await Twitter.filter(asa_id=asaID)
             .annotate(
                 sentimentTotal=Avg("sentiment_score"),
-                retweetTotal=Sum("retweets"),
-                likeTotal=Sum("likes"),
+                retweetTotal=Sum("retweets")/Count("retweets"),
```

Comment:
This computation is done on the server. Use Avg to perform it on the backend, as I did for the sentiment score.

Reply:
Oh yeah, got that!